### ISSUE TITLE
[LINT_BUG]: ﻿﻿﻿﻿box_usage_linter functions defined with {zeallot}'s `%<-%` aren't recognized

### ISSUE THREAD

**Issue by TymekDev** [Appsilon/box.linters#156]
### box.linters version

0.10.5

### Sample source code to lint

```r
box::use(
  zeallot[`%<-%`],
)

c(reactive1, reactive2) %<-% list(function() 11, function() 12)

print(reactive1() + reactive2())
```

### Lint command used

```r
lintr::lint(
  "box::use(
  zeallot[`%<-%`],
)

c(reactive1, reactive2) %<-% list(function() 11, function() 12)

print(reactive1() + reactive2())",
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)
```

### Lint result

```
﻿<text>:7:7: warning: [box_usage_linter] Function not imported nor defined.
print(reactive1() + reactive2())
      ^~~~~~~~~
<text>:7:21: warning: [box_usage_linter] Function not imported nor defined.
print(reactive1() + reactive2())
```

### Expected result

No lints raised. `%<-%` is recognized as an assignment operator.

**Comment by radbasa** [comment]
@TymekDev do you have time to try this again with the current `main` branch?

**Comment by TymekDev** [comment]
Looks good now, thanks!

![](https://github.com/user-attachments/assets/b86ebc6d-aa30-4962-a834-6dbb07b2255a)
